src/include/binder/copy/bound_copy.h

```diff
@@ -11,25 +11,55 @@
 namespace kuzu {
 namespace binder {
 
+struct BoundCopyInfo {
+    common::table_id_t tableID;
+    std::string tableName;
+};
+
 class BoundCopy : public BoundStatement {
 public:
-    BoundCopy(
-        common::CopyDescription copyDescription, common::table_id_t tableID, std::string tableName)
-        : BoundStatement{common::StatementType::COPY,
-              BoundStatementResult::createSingleStringColumnResult()},
+    // COPY FROM
+    BoundCopy(common::CopyDescription copyDescription, common::table_id_t tableID,
```

Comment:
Split BoundCopy to BoundCopyFrom and BoundCopyTo. BoundCopyFrom should stay the same as it used to be.

BoundCopyTo should have
```
common::CopyDescription copyDescription;
std::unique_ptr<BoundStatement> regularQuery; // or just call it statement
```